server/src/sim.rs

```diff
@@ -370,6 +386,15 @@ impl Sim {
             self.dirty_nodes.insert(position.node);
         }
 
+        for (position, blinker) in pending_blinker_spawns {
```

Comment:
One of the changes I made in my PR provided a more succinct way of accomplishing this:
```rust
for (position, blinker) in pending_blinker_spawns {
    self.spawn((position, blinker));
}
```

Reply:
Cool, thanks!